shared/validation/languages/it.js

```diff
@@ -10,9 +10,9 @@ const NUMBERS_REGEX = /[0-9]+/;
 
 /* eslint-disable-next-line no-useless-escape */
 // Italian: Simboli non permessi, aggiungere anche qui sotto oltre che nella regex:
-// < > + * \ # @ ^ “ ” ‘ ’ ( ) É [ ] /
+// < > + * \ # @ ^ “ ” ‘ ’ ( ) É [ ] / { }
 //doppio " " e più di un "." nella stessa frase.
-const SYMBOL_REGEX = /[<>+*\\#@^“”‘’()É[\]/]|\s{2,}|\..*\./;
+const SYMBOL_REGEX = /[<>+*\\#@^“”‘’()É[\]/]|\s{2,}|\..*\./{};
```

Comment:
this won't work, as like this `{}` is not part of the Regex defined through the `/ ... / ` and leads to a syntax error.